graph-builder/tests/net/quay_io/mod.rs

```diff
@@ -131,6 +152,7 @@ fn fetch_release_private_with_credentials_must_succeed() {
             password.as_ref().map(String::as_ref),
             &mut cache,
             MANIFESTREF_KEY,
+            8,
```

Comment:
Lets use a `static` for this magic number (preferably defined in `graph-builder/src/config/options.rs`)